pygmt/src/basemap.py

```diff
@@ -0,0 +1,78 @@
+"""
+basemap - Plot a base map and frames.
+"""
```

Comment:
See line 36 of this file. It would be better if the function description is the same as the one shown in the API reference (i.e., line 36).

Reply:
Copied line 36.